jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskdetailsmulti/TaskDetailsMultiViewImpl.java

```diff
@@ -225,9 +234,15 @@ public void displayOnlyLogTab() {
         for ( Widget active : tabContent ) {
             active.setVisible( false );
         }
+        taskDetailsPane.setVisible(true);
+        taskDetailsTab.setVisible(true);
+        presenter.disableTaskDetailsEdition();
```

Comment:
here is looping back to the presenter, instead I think you could this in the presenter itself right after calling view.displayOnlyLogTab ?